pallets/proposals/src/tests.rs

```diff
@@ -1452,10 +1465,29 @@ fn test_project_initiator_can_withdraw_only_the_percentage_after_force_milestone
             mock::RuntimeEvent::from(proposals::Event::ProjectFundsWithdrawn(
                 *ALICE,
                 0,
-                500000u64,
+                deduct_imbue_fee(500_000u64),
                 CurrencyId::Native
             ))
         );
+
+        assert_eq!(
+            Tokens::free_balance(CurrencyId::Native, &pallet_account),
+            account_balance.saturating_add(calc_imbue_fee(contribution_value))
+        );
+
+        //calculating the total percentage that can be withdrawn based on the submitted milestones
+        let total_percentage_to_withdraw: u32 =
+            proposed_milestones1.get(0).unwrap().percentage_to_unlock
+                + proposed_milestones1.get(1).unwrap().percentage_to_unlock;
+
+        //making sure that only balance is equal to the amount withdrawn
+        //making sure not all the required funds have been assigned instead only the percentage eligible could be withdrawn
+        let withdrawal_amount =
+            contribution_value.saturating_mul(2) * (total_percentage_to_withdraw as u64) / 100;
```

Comment:
this looks odd, it should be the project.raised_funds * percent/100;
I guess it works as the contribution amounts are equal? + there are only 2